src/eip4824.ts

```diff
@@ -82,6 +82,13 @@ router.get('/:space/proposals', async (req, res) => {
     type: 'proposal',
     id: proposal.id,
     name: proposal.title,
+    contentURI: {
+      body: proposal.body,
+      author: proposal.author,
+      ipfs: proposal.ipfs,
+      link: proposal.link
```

Comment:
There is no link field on proposal 🤔 what should be the value

Reply:
The snapshot URL of the proposal, we have this on the hub API

Reply:
Link of the Snapshot Proposal

`"link": "https://snapshot.org/#/opcollective.eth/proposal/0xc9c0a739a29ce6477708f49a2985e4c8cb2e4ba0be1d3761541ed189a9c341e1",`

I found this from GraphQL API,

Reply:
@Rashmi-278 no access for me to change it,

Could you change these:
- Remove `link` from the above query
- You can use the same logic as https://github.com/metagov/snapshot-hub/blob/pr/Rashmi-278/911/src/graphql/helpers.ts#L401-L402

Reply:
I face 404 on the above link, do you mean to remove link from query and `Query *` ?